### PR TITLE
Add REPL section

### LINKED ISSUE #147: Add an embeddable REPL powered by JupyterLite.
I intend to look into adding an interactive REPL section similar to one deployed on the [NumPy website](https://numpy.org/) to our landing page. 

Screen Shot 2022-02-23 at 10 06 48 AM

This interactive REPL would use jupyterlite and would run xarray entirely in the browser

### PR DESCRIPTION
**Pull request by andersy005**
- [x] Closes #147 

Screenshot from 2022-02-27 22-58-38

### CODE REVIEW

**Comment by jhamman** [comment]
So cool!

File: src/components/ReplSection.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  Heading,
+  Text,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react"
+
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter"
+import { a11yDark } from "react-syntax-highlighter/dist/cjs/styles/prism"
+
+//TODO: We could remove xarrray installation step by requesting xarray to be added to the Pyodide standard lib
```
Comment (jhamman) [inline src/components/ReplSection.js:15]: This seems like a reasonable thing to ask? What's the process of adding a package to the standard lib?

File: src/components/ReplSection.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+//TODO: Or wait until jupyterlite has streamlined the installation process of user-specified packages
```
Comment (jhamman) [inline src/components/ReplSection.js:16]: I think this is useful enough to warrant the small inconvenience of running micropip here.

**Review by jhamman — APPROVED** [review]

File: src/components/ReplSection.js (hunk 1 above)
Comment (andersy005) [reply src/components/ReplSection.js:15]: we need to open an issue/pr in https://github.com/pyodide/pyodide

**Pull request merged**